Docker-README.md

```diff
@@ -19,6 +19,7 @@ Other data involved in the Docker build process can be found at https://github.c
 
 The images are ready to run with limited functionality.
 At container startup, the startup.py wrapper (from the dockerdata directory linked above) checks for `PDNS_RECURSOR_API_KEY` / `PDNS_AUTH_API_KEY` / `DNSDIST_API_KEY` environment variables for the product you are running.
+You can also use `PDNS_RECURSOR_API_KEY_FILE` / `PDNS_AUTH_API_KEY_FILE` / `DNSDIST_API_KEY_FILE` variables for Docker's secret.
```

Comment:
This text sound like Docker has some feature for secrets, but this PR just adds support for reading the keys from a file. Or am I missing something?

Is this https://docs.docker.com/engine/swarm/secrets/ ? If so, can you add a reference to that, and perhaps point out some usual values for the _FILE vars? Thanks!